about.html

```diff
@@ -0,0 +1,196 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link
+			rel="stylesheet"
+			href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+		/>
+    <title>Document</title>
+</head>
+<body>
+    <header>
+        <div class="ham-icon">
+            <button class="ham-btn" type="button">
+                <span class="ham-span-btn"></span>
+                <span class="ham-span-btn"></span>
+                <span class="ham-span-btn"></span>
+            </button>
+        </div>
+            <nav class="mobile-nav-menu">
+                <ul>
+                    <li>
+                        <a href="index.html">HOME</a>
+                    </li>
+                    <li>
+                        <a href="#">PROGRAM</a>
+                    </li>
+                    <li>
+                        <a href="#">SPONSORS</a>
+                    </li>
+                    <li>
+                        <a href="#">SPEAKERS</a>
+                    </li>
+                </ul>
+            </nav>
+            <div class="desk-nav-bar">
+                <nav class="desk-nav-one">
```

Comment:
- [ ] It seems that when you implemented some of the changes requested by the previous reviewer, something went bad on your navbar. Now, your navbar doesn't fit all the width as requested. please, kindly fix this issue. you can have a look here below:
![image](https://user-images.githubusercontent.com/71874425/170838689-cdb01dd1-c068-4917-86be-a5493fbfef88.png)